software-templates/techdocs-template/skeleton/README.md

```diff
@@ -0,0 +1,63 @@
+[![Package Status](https://${{values.destination.host}}/${{values.destination.owner}}/${{values.destination.repo}}/actions/workflows/publish.yaml/badge.svg)](https://${{values.destination.host}}/${{values.destination.owner}}/${{values.destination.repo}}/actions/workflows/publish.yaml)
+
+
+# ${{values.name}}
+
+${{values.description}}
+
+## About
+
+This is a default [README.md](README.md) file generated by the template. Customize it for your project.
+
+Refer to the [index.md](docs/index.md) file for instructions on how to add documentation.
+
+
+## Setup
+
+Replace this section with instructions to set up your project locally. 
+### Prerequisites
+
+Use this section to list any prerequisites needed to install your project. Example:
+* npm
+  ```sh
+  npm install npm@latest -g
+  ```
+
+### Installation
+
+List the steps to installs your project. For example:
+
+1. Clone the repo
+   ```sh
+   git clone https://${{values.destination.host}}/${{values.destination.owner}}/${{values.destination.repo}}.git
+   ```
+1. Create a `config.local.yaml` file
+1. Set the `env` variable to `local` in the `config.local.yaml` file
+   ```yaml
+   env: local
+   ```
+
+### Usage
+
+Use this section to describe how to use your project. Example:
+
+```sh
+yarn start
+```
+
+## Getting Help or Reporting an Issue
+
+To report bugs/issues/features requests, please file an [issue]( https://${{values.destination.host}}/${{values.destination.owner}}/${{values.destination.repo}}/issues).
+
```

Comment:
Also wondering if the above Setup section is too much for a techdocs only repo? Maybe the above is more appropriate for a code repo?

Reply:
Maybe link to https://github.com/bcgov/devhub-techdocs-publish/blob/main/docs/index.md instead on how to view techdocs locally?